sw/device/tests/rstmgr_cpu_info_test.c

```diff
@@ -108,17 +106,26 @@ static void read_error(void) {
 void ottf_exception_handler(void) {
   uint32_t addr = (uint32_t)exp_dump;
 
-  temp_dump[kCpuDumpIdxCurrentExceptionAddr] =
+  // the exception address ends up being the same since both are
```

Comment:
Comments should be full sentences (Start with Capital letter and finish with period).

Reply:
fixed!